build-support/bin/check_header_helper.py

```diff
@@ -8,8 +8,8 @@
 #
```

Comment:
This file has the only semantic changes. Updates the header checker to enforce this new simplified header.